tests/spread/plugins/craft-parts/build-and-run-hello/matter-hello/snap/snapcraft.yaml

```diff
@@ -0,0 +1,54 @@
+name: matter-hello
+version: "1.0"
+summary: simple matter application
+description: build an example lighting application using Matter SDK and say "hello world"
+base: core22
+confinement: strict
+
+apps:
+  lighting:
+    daemon: simple
+    install-mode: disable
+    command: bin/lighting-app
+    plugs:
+      - network
+      - network-bind
+      - bluez
+      - avahi-control
+      - gpio-memory-control
+      - otbr-dbus-wpan0
+
+parts:
+  lighting-hello:
+    plugin: matter
+    matter-sdk-version: "master"
+    matter-zap-version: "v2023.11.13"
+    source: app
+    override-build: |
+      craftctl default
+      cd $CRAFT_PART_BUILD/matter
+
+      # Build a lighting example
+
+      cd examples/lighting-app/linux
+      gn gen out/build --args='is_debug=true'
+      ninja -C out/build
```

Comment:
Building error:
```
:: [48/595] ACTION //third_party/connectedhomeip/examples/lighting-app/lighting-common:lighting-common_codegen_codegen(//third_party/connectedhomeip/build/toolchain/linux:linux_x64_gcc)                                                                                                                                                                                  
:: FAILED: gen/third_party/connectedhomeip/examples/lighting-app/lighting-common/app/PluginApplicationCallbacks.h gen/third_party/connectedhomeip/examples/lighting-app/lighting-common/app/callback-stub.cpp                                                                                                                                                              
:: python ../../third_party/connectedhomeip/third_party/pigweed/repo/pw_build/py/pw_build/python_runner.py --gn-root ../../ --current-path ../../third_party/connectedhomeip/examples/lighting-app/lighting-common --default-toolchain=//third_party/connectedhomeip/build/toolchain/linux:linux_x64_gcc --current-toolchain=//third_party/connectedhomeip/build/toolchain/linux:linux_x64_gcc --capture-output --python-dep-list-files gen/third_party/connectedhomeip/examples/lighting-app/lighting-common/lighting-common_codegen_codegen_metadata_path_list.txt -- ../../third_party/connectedhomeip/scripts/codegen.py --generator cpp-app --output-dir gen/third_party/connectedhomeip/examples/lighting-app/lighting-common --expected-outputs gen/third_party/connectedhomeip/examples/lighting-app/lighting-common/lighting-common_codegen.expected.outputs ../../third_party/connectedhomeip/examples/lighting-app/lighting-common/lighting-app.matter                                                                                                                                                                  
:: Traceback (most recent call last):                                                                                                                                                
::   File "/root/parts/lighting-hello/build/matter/examples/lighting-app/linux/out/build/../../third_party/connectedhomeip/scripts/codegen.py", line 19, in <module>                 
::     import click                                                                                                                                                                  
:: ModuleNotFoundError: No module named 'click'
```

Reply:
The error is caused by an incomplete bootstrap. Adding the following commands solves the issue:
```
set +u && source scripts/setup/bootstrap.sh --platform build && set -u
```

Reply:
With a clean build environement, chip-tool using Matter plugin doesn't require extra bootstrap.